src/ccnl-utils/src/ccn-lite-rpc.c

```diff
@@ -365,6 +369,9 @@ main(int argc, char *argv[])
 
         nonce = calloc(1, sizeof(*nonce));
         if (!nonce) {
+            if (expr) {
```

Comment:
here also; is the check to `expre` necessary, given that it is checked above?

Reply:
Okay, I can remove the check. Putting a return -1 and omitting the curly brackets and then putting them there afterwards indicating that the if statement is somehow related to the following '{' seems a bit more concerning to me.